canvas_modules/common-canvas/package.json

```diff
@@ -108,7 +108,7 @@
     "react": "^16.0.0 || ^17.0.0 || ^18.0.0",
     "react-dom": "^16.0.0 || ^17.0.0 || ^18.0.0",
     "react-intl": "^5.0.0 || ^6.0.0",
-    "react-redux": "^7.0.0"
+    "react-redux": "^7.0.0 || ^8.0.0"
```

Comment:
I think we need to remove `react-redux` from peerDependencies and move it from dev to prod dependencies. This way common-canvas can have it's own version.